prow/cmd/mkpod/main.go

```diff
@@ -82,13 +94,41 @@ func main() {
 		o.buildID = job.Status.BuildID
 	}
 
+	if strings.ToLower(o.buildID) == "snowflake" {
```

Comment:
Why not just let the user provide one and not use the special case here?

Reply:
We already let the user provide one if they want something specific, this doesn't change that. Most of the time the user doesn't care about the build ID though and just wants a unique one so forcing the user to provide a unique ID each time sounds unreasonable to me especially when we default to generating snowflake IDs in prod which is something mkpod can totally do.